cmd/context/create_test.go

```diff
@@ -426,6 +426,7 @@ func TestCheckAccessToNamespace(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		tt := tt
```

Comment:
@jLopezbarb the tests seem to be failing because of this line. It's unrelated to the PR and added it just because I saw the `loop variable tt captured by func literal` warning. I removed it but we might wanna look into it

Reply:
True, I think it's because we can't run parallel tests using environment variables. I'll take a look and find a proper fix for this